pkg/settings/settings.go

```diff
@@ -86,6 +86,7 @@ const (
 	SupportBundleExpirationSettingName                = "support-bundle-expiration"
 	NTPServersSettingName                             = "ntp-servers"
 	AutoRotateRKE2CertsSettingName                    = "auto-rotate-rke2-certs"
+	KubeconfigTTLSettingName                          = "kubeconfig-default-token-ttl-minutes"
```

Comment:
I would suggest to use this name because it is more clear what it is about and it matches the naming schema of all other constants defined in that file.
```suggestion
	KubeconfigDefaultTokenTTLMinutesSettingName                          = "kubeconfig-default-token-ttl-minutes"
```